Purchases/Purchasing/IntroEligibilityCalculator.swift

```diff
@@ -30,6 +30,7 @@ class IntroEligibilityCalculator {
     func checkEligibility(with receiptData: Data,
                           productIdentifiers candidateProductIdentifiers: Set<String>,
                           completion: @escaping ([String: IntroEligibilityStatus], Error?) -> Void) {
+
```

Comment:
Nit: we only add spaces to types not functions

Reply:
GitHub needs a 🤦‍♂️ reaction